test/JDBC/expected/BABEL-2844.out

```diff
@@ -135,6 +165,11 @@ Seq Scan on babel_2844_t1  (cost=0.00..38.25 rows=11 width=8)
   Filter: (b1 = 1)
 ~~END~~
 
+~~START~~
+xml
+<explain xmlns="http://www.postgresql.org/2009/explain"><newline>  <Babelfish-T-SQL-Batch-Parsing-Time>0.109</Babelfish-T-SQL-Batch-Parsing-Time><newline></explain>
```

Comment:
When the original plan is not XML format, why the tsql parsing time is XML format ?

Reply:
seems like this is existing bug with BABELFISH_SHOWPLAN_ALL. Seems like it does not obey setting of `babelfishpg_tsql.explain_format`. I will file separate bug to fix it.

Reply:
EXPLAIN does not follow setting of babelfishpg_tsql.explain_format. We will fix it separately